services/device-quota-suggestion-service/tests/test_api.py

```
@@ -0,0 +1,108 @@
+from fastapi.testclient import TestClient
+
+from app.embeddings import DeterministicEmbeddingBackend
+from app.main import create_app
+from app.settings import Settings
+
+
+def build_client():
+    app = create_app(
+        settings=Settings(internal_token="test-token"),
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to argument: "internal_token"

(S106)